readthedocs/vcs_support/backends/svn.py

```diff
@@ -91,7 +91,9 @@ def parse_tags(self, data):
             release-1.5/
         """
         # parse the lines into a list of tuples (commit-hash, tag ref name)
-
+        # StringIO below is expecting Unicode data, so ensure that it gets it.
+        if not isinstance(data, str):
+            data = str(data)
```

Comment:
```
+from future import standard_library
 +standard_library.install_aliases()
```
basically makes `str` on Python 2 act like `unicode` and adds `bytes` as the equivalent to the original `str`.